crimsobot/cogs/utilities.py

```diff
@@ -36,11 +36,16 @@ async def poll(self, ctx: commands.Context, *, poll_input: str) -> None:
 
         choices = poll_input.split(';')
         question = choices.pop(0).strip()
+        attachment = None
 
         # some tests to ensure there's a question and at least two choices
         if len(question) == 0:
```

Comment:
Unsure why this snippet got past. This can be shifted to use `not` (`if not question`) for cleanliness.

Reply:
Not sure why I put that down either. Removed.